src/java.base/unix/classes/java/lang/ProcessEnvironment.java

```diff
@@ -52,37 +52,96 @@
  * @since   1.5
  */
 
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2022, 2022 All Rights Reserved
+ * ===========================================================================
+ */
+
 package java.lang;
 
 import java.io.*;
 import java.util.*;
-
+import openj9.internal.criu.InternalCRIUSupport;
```

Comment:
if we dont have pre-processor support we'll need to make `InternalCRIUSupport` enabled in all builds

Reply:
Agreed that `InternalCRIUSupport` should be enabled by default, otherwise security JCL patches are going to have same problem.

Reply:
CRIU APIs are available to JDK11+ builds via `ServiceLoader.load(CRIUProvider.class)` - https://github.com/eclipse-openj9/openj9/pull/14730